JSDemos/Demos/TreeView/HierarchicalDataStructure/description.md

```diff
@@ -1 +1,16 @@
-The TreeView component displays a&nbsp;hierarchical collection of&nbsp;items using a&nbsp;traditional tree structure. TreeView nodes can be&nbsp;expanded and collapsed to&nbsp;display sub-items.
\ No newline at end of file
+A hierarchical array contains items that can nest other items. To bind the TreeView to such a local array, assign it to the [items[]](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/) property as shown in this demo. If you want to process the array or bind the TreeView to remote data, use the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object and specify the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataSource) property instead of **items[]**.
```

Comment:
"Such a local array" reads as "the array is sooooo local" :)

```suggestion
A hierarchical array contains items that can nest other items. To bind the TreeView to a local hierarchical array, assign it to the [items[]](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/) property as shown in this demo. If you want to process the array or bind the TreeView to remote data, use the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object and specify the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataSource) property instead of **items[]**.
```